style/rustfmt.toml

```diff
@@ -0,0 +1,8 @@
+error_on_unformatted = true
+imports_granularity = "Crate"
+tab_spaces = 4
+hard_tabs = false
+indent_style = "Block"
+max_width = 100
```

Comment:
Change max_width to 150 to match other formatting files

Reply:
A bit hesitant on this. Many IDEs such as CLion realistically only support 100 characters of width before it goes off screen.